packages/dev-utils/api-report.md

```diff
@@ -16,6 +16,7 @@ import { PropsWithChildren } from 'react';
 import { default as React_2 } from 'react';
 import { ReactNode } from 'react';
 import { SignInProviderConfig } from '@backstage/core-components';
+import { TranslationResource } from '@backstage/core-plugin-api';
```

Comment:
feel free to import these from `@backstage/core-plugin-api/alpha` for now

Reply:
Works for me! https://github.com/backstage/backstage/compare/dev_app_translations?expand=1 This at least runs tests and api reports fine, I think?

Reply:
Weird, I thought I got some warning to API reports from that. Fix'd now.